documentation/specs/src/base-ledger/governance.md

```diff
@@ -16,18 +16,18 @@ The second internal address holds the funds of rejected proposals.
 Each proposal will be stored in a sub-key under the internal proposal address. The storage keys involved are:
 
 ```
-/\$GovernanceAddress/proposal/$id/content: Vec<u8>
-/\$GovernanceAddress/proposal/$id/author: Address
-/\$GovernanceAddress/proposal/$id/start_epoch: Epoch
-/\$GovernanceAddress/proposal/$id/end_epoch: Epoch
-/\$GovernanceAddress/proposal/$id/grace_epoch: Epoch
-/\$GovernanceAddress/proposal/$id/proposal_code: Option<Vec<u8>>
-/\$GovernanceAddress/proposal/$id/funds: u64
-/\$GovernanceAddress/proposal/epoch/$id: u64
+/\$GovernanceAddress/proposal/\$id/content: Vec<u8>
```

Comment:
the katex pre-processer's rules seem pretty odd, but when I try to render this page (`make serve`) locally, in this block and couple other places below I actually see `/\` rendered, so it seems that these ones don't need escaping